docs/getting-started-guides/README.md

```diff
@@ -8,6 +8,7 @@ GKE            |              |        | GCE         | [docs](https://cloud.goog
 AWS            | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community                    | Uses K8s version 0.11.0
 GCE            | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community (@kelseyhightower) | Uses K8s version 0.11.0
 Vagrant        | CoreOS       | CoreOS | flannel     | [docs](../../docs/getting-started-guides/coreos.md)    | Community (@pires)           | Uses K8s version 0.11.0
+Vagrant        | CoreOS       | CoreOS | flannel     | [docs](https://github.com/AntonioMeireles/kubernetes-vagrant-coreos-cluster/blob/master/README.md)    | Community (@AntonioMeireles)           | Uses K8s version 0.13.2
```

Comment:
I don't think this is meant to point to external repos but rather to `getting-started-guides` folder stuff. I for one would like you to keep updating it as you have been doing in the past. You should replace my name.